<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rollenspielhimmel Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="MainWindowLayout">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 237, 223)</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="OptionWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="OptionLayout" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="inputUrlLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;</string>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="placeholderText">
            <string>url</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="onlyOnePageCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Nur eine Seite</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="lastPageLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="firstPageLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Erste Seite</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="firstPageSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="firsPageLayout">
           <property name="topMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="lastPageLabel">
             <property name="text">
              <string>Letzte Seite</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="lastPageSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="getPostsActionButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 10;
height: 30px;
</string>
           </property>
           <property name="text">
            <string>Posts holen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="postLoadProgressBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="eeveeMovie">
         <property name="text">
          <string>eevee.gif</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="copyrightEeveeMovie">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>ArtÂ©:TheDarkShadow123</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="PostsBoxLayout" stretch="0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QListView" name="postsListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="loginLayout" stretch="0,0,0,0">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginActionButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="statusLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="loginStatusLabel">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="menubarSaveAction"/>
    <addaction name="separator"/>
    <addaction name="menubarCloseAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="menubarDeleteAction"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="menubarSaveAction">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="menubarCloseAction">
   <property name="text">
    <string>SchlieÃen</string>
   </property>
  </action>
  <action name="menubarDeleteAction">
   <property name="text">
    <string>LÃ¶schen</string>
   </property>
  </action>
  <action name="menubarLoginAction">
   <property name="text">
    <string>Einloggen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
